### PR TITLE
Only process sourceMappingURLs for files, not paths

### PR DESCRIPTION
**Pull request by dhh**
This will prevent the recursion issue noticed in #481.

### CODE REVIEW

File: test/test_sourcemapping_url_processor.rb [hunk 1]
```diff
@@ -24,6 +24,16 @@ def asset_path(path, options = {})
     assert_equal({ data: "var mapped;\n//# sourceMappingURL=mapped-HEXGOESHERE.js.map\n//!\n" }, output)
   end
 
+  def test_prevent_recursion
+    input = { environment: @env, data: "var mapped;\n//# sourceMappingURL=/assets/mapped.js.map", filename: 'mapped.js', metadata: {} }
```
Comment (rafaelfranca) [inline test/test_sourcemapping_url_processor.rb:28]: Source maps inside directories are not supported?

Comment (dhh) [reply test/test_sourcemapping_url_processor.rb:28]: Currently not.

**Comment by dhh** [comment]
This wasn't it.

File: test/test_sourcemapping_url_processor.rb (hunk 1 above)
Comment (dhh) [reply test/test_sourcemapping_url_processor.rb:28]: Managed to add support in this new PR I'm working on.

**Pull request closed without merging**